labs/lab18-helm-charts/README.md

```diff
@@ -0,0 +1,435 @@
+
+```bash
+git clone https://github.com/bitnami/charts.git
+cd charts
+git fetch origin 5277d698528bc4a78389f83c7e6fa4c42a6e63ad
+git reset --hard 5277d698528bc4a78389f83c7e6fa4c42a6e63ad
+```
+
+The above commands should result in a message similar to:
+
+```
+HEAD is now at 5277d6985 [bitnami/wordpress] Release 14.0.7 (#10066)
+```
+
+This command will have downloaded all the bitnami helm chart files. In order to include a Helm Chart in Replicated, we have to package it first.
+
+### 2. Change Directory to the Wordpress Chart
+
+As mentioned above, this repo contains several Helm Charts, and we want to only package WordPress, so we need to change directory:
+
+```bash
+cd bitnami/wordpress/
+```
+
+We are now at the root of the Wordpress Helm Chart. If you list the contents of this directory, you will see that it contains a `Chart.yaml` and `Values.yaml` file. The `Chart.yaml` contains details about the chart, and the `Values.yaml` contains fields that the end user can override with their own values. Fields in this file can also be passed as parameters when you run `helm install`.
+
+Let's verify that we are using the correct version of the chart by looking at the contents:
+```bash
+cat Chart.yaml
+```
+
+At the end of the file we should see the following:
+
+```bash
+...
+name: wordpress
+sources:
+  - https://github.com/bitnami/bitnami-docker-wordpress
+  - https://wordpress.org/
+version: 14.0.7
+```
+
+### 3. Package the Helm Chart
+
+To package the chart, we are going to run the `helm package` command.
+
+As mentioned above, this Helm Chart includes some dependencies. To ensure that these are included in this Chart, include the `-u` option. Also note that we are passing a `.` for the location of the chart. This argument indicates the path to a directory containing the top level ```Chart.yaml``` file. For more details on `helm package`, please refer to the [Helm Documentation](https://helm.sh/docs/helm/helm_package/#helm).
+
+```bash
+helm package -u .
+```
+The command may display some warnings, but ultimately should display an output similar to
+
+```
+Successfully packaged chart and saved it to: /Users/fernandocremer/test/tests/charts/bitnami/wordpress/wordpress-14.0.7.tgz
+
+```
+
+We will include this file in our Replicated Application Release.
+
+## Adding the Helm Chart to a Release
+
+You should have received an invite to log into https://vendor.replicated.com -- you'll want to accept this invite and set your password. 
+
+In this lab, we have already created an application for you, but there are no releases created yet. We are going to cover creating the release with the UI and through the CLI. Going through the UI is easier and better if you just want to understand the basic concepts as it will automatically create resources for you. With the CLI, we'll take a more 'starting from scratch' approach. Later in the lab we'll be making changes to the application, and will only cover the CLI.
```

Comment:
```suggestion
In this lab, an application has already been created but there are no releases yet. We are going to cover creating the release with the UI and through the CLI. Attaching your chart through the UI is easier and better if you just want to understand the basic concepts as it will automatically create resources for you. Later in this lab, we'll cover using the Replicated CLI to create and update releases in a helm-based application.
```